src/detect/ScanI2CTwoWire.cpp

```diff
@@ -292,7 +292,18 @@ void ScanI2CTwoWire::scanPort(I2CPort port)
 
                 break;
 
-                SCAN_SIMPLE_CASE(SHT31_ADDR, SHT31, "SHT31 sensor found\n")
+            case SHT31_4x_ADDR:
+                registerValue = getRegisterValue(ScanI2CTwoWire::RegisterLocation(addr, 0x89), 2);
+                if (registerValue == 0x11a2) {
+                    type = SHT4X;
+                    LOG_INFO("SHT4X sensor found\n");
+                } else {
+                    type = SHT31;
+                    LOG_INFO("SHT31 accelerometer found\n");
```

Comment:
Copy paste issue? Should say sensor instead of accelerometer

Reply:
indeed, fixed. Thank you